ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/BlockImporter.java

```diff
@@ -81,4 +81,20 @@ BlockImportResult importBlockForSyncing(
       HeaderValidationMode headerValidationMode,
```

Comment:
This old importBlockForSyncing should be able to be removed now

Reply:
Done